travvy/src/components/Onboarding.vue

```diff
@@ -60,24 +83,50 @@ export default {
     msg: String
   }, 
   
+  data() {
+    return {
+                email: '',
+                age: '',
+                gender: '',
+                instagram_username: '',
+                preferred_activity_type: '',
+                error: null,
+            }
+
+  },
   methods:{
-    async signup(){
-      console.log(this.name)
+    async onboarding(){
       console.log(this.email)
-      console.log(this.password)
-                
+      console.log(this.age)
+      console.log(this.gender)
+      console.log(this.instagram_username)
+      console.log(this.preferred_activity_type)
```

Comment:
Maybe these logs can be removed for cleaner code

Reply:
Thats a good idea! I added those when i was testing the functionality of the function